options/locale/locale_en-US.ini

```diff
@@ -1372,6 +1372,19 @@ settings.transfer_desc = Transfer this repository to a user or to an organizatio
 settings.transfer_notices_1 = - You will lose access to the repository if you transfer it to an individual user.
 settings.transfer_notices_2 = - You will keep access to the repository if you transfer it to an organization that you (co-)own.
 settings.transfer_form_title = Enter the repository name as confirmation:
+settings.signing_settings = Signing Verification Settings
+settings.trust_model = Signature Trust Model
+settings.trust_model.default = Default Trust Model
+settings.trust_model.default.desc= Use the default repository trust model for this installation.
+settings.trust_model.collaborator = Collaborator
+settings.trust_model.collaborator.long = Collaborator: Trust signatures signed by keys of collaborators
+settings.trust_model.collaborator.desc = Signatures from keys owned by collaborators of this repository will be trusted - (whether they match the Committer or not). Otherwise, commits will be marked untrusted if the signature matches the committer and unmatched if the signature otherwise verifies.
+settings.trust_model.committer = Committer
+settings.trust_model.committer.long = Committer: Trust signatures that match committers (This matches GitHub and will force Gitea signed commits to have Gitea as the commmitter)
+settings.trust_model.committer.desc = Signatures will only be marked trusted if they match the committer. If they otherwise verify they will be marked unmatched. This will force Gitea to be marked as the committer on signed commits with the actual committer marked as Co-Authored-By: and Co-Committed-By: trailer in the commit. The default Gitea key must match a User in the database,
```

Comment:
```suggestion
settings.trust_model.committer.desc = Signatures will only be marked as trusted if they match the committer, otherwise they will be marked as unmatched if signature is valid. This will force Gitea to be committer for signed CRUD actions with the actual committer marked as Co-Authored-By: and Co-Committed-By: trailer in the commit. The default Gitea key must match a User in the database.
```